python/lsst/pipe/tasks/calibrateImage.py

```diff
@@ -658,7 +653,6 @@ def run(self, *, exposures, id_generator=None, result=None):
             result.psf_stars_footprints, result.background, _ = self._compute_psf(result.exposure,
                                                                                   id_generator)
             have_fit_psf = True
-            summary_stat_catalog = result.psf_stars_footprints
```

Comment:
From looking at the current version of the code, my understanding was that this was set here in case the next task failed and it bumped out to the except block.  Is this no longer needed?

Reply:
I was originally trying to have the summary statistics calculations (specifically the PSF ones, which are the only ones that use a catalog at all) use the initial set of detections and measurements when the second round wasn't available due to a failure.  But it turns out it doesn't have the right columns for that to work, and I didn't think it was worth it to add more columns to the initial round or add more logic to the summary stats to make it work.  There just aren't enough failures in the particular window where this fallback is relevant (it's basically just aperture correction measurement failures) to be worth the special-casing to get a little bit more summary stat info out on detectors we can't use downstream anyway.